modynclient/client/client.py

```diff
@@ -169,3 +170,24 @@ def poll_pipeline_status(self) -> None:
             logger.info(f"Pipeline <{self.pipeline_id}> not found.")
         else:
             logger.error(f"unknown pipeline status {json.dumps(res, sort_keys=True, indent=2)}")
+
+
+def run_multiple_pipelines(
+    client_config: ModynClientConfig,
+    pipeline_configs: list[ModynPipelineConfig],
+    start_replay_at: int = 0,
+    stop_replay_at: int | None = None,
+    maximum_triggers: int | None = None,
+) -> None:
+    logger.info("Start running multiple experiments!")
+    logger.info(f"{client_config.model_dump()}")
+
+    for pipeline_config in pipeline_configs:
+        client = Client(client_config, pipeline_config, start_replay_at, stop_replay_at, maximum_triggers)
+
+    logger.info(f"Starting pipeline: {pipeline_config.pipeline.name}")
+    started = client.start_pipeline()
+    if started:
+        client.poll_pipeline_status()
```

Comment:
Should this be indented one level further? Because now you instantiate a lot of clients (overriding the variable) and only start one. Maybe it's good to have one test run of this code since you mention you didnt check it yet 😅

Reply:
Just discovered that a couple of minutes ago on my local branch when running kick the script  :)